frictionless/program/validate.py

```diff
@@ -11,6 +13,31 @@
 from . import common
 
 
+def _wrap_text_to_colwidths(
```

Comment:
Please put internal code to the bottom of the file so a reader can see more important functions first.
Also, I think we need to add a #NOTE that we're waiting a new tabulate release to remove this func

Reply:
I see you added it to the docstring but it's also good to have `#TODO or #NOTE` to make it searchable